src/Confluent.Kafka/IAdminClient.cs

```diff
@@ -531,4 +531,77 @@ public interface IAdminClient : IClient
         Task AlterUserScramCredentialsAsync(IEnumerable<UserScramCredentialAlteration> alterations, AlterUserScramCredentialsOptions options = null);
     }
 
+    /// <summary>
+    ///     Extension methods for default <see cref="IAdminClient"/> implementations.
+    /// </summary>
+    public static class IAdminClientExtensions
```

Comment:
I see we are doing this to avoid breaking binary compatibility but we don't guarantee that for .NET, right? 
Also, as per https://learn.microsoft.com/en-us/dotnet/core/compatibility/categories#binary-compatibility
If we just add methods or method implementations, that doesn't break binary compatibility, too, right?

Reply:
Yeah, it's extending an interface that breaks it, we're starting to avoid breaking it.